editions/tw5.com/tiddlers/howtos/How to embed PDF and other documents.tid

```diff
@@ -4,32 +4,32 @@ tags: Learning
 title: How to embed PDF and other documents
 type: text/vnd.tiddlywiki
 
-!!! Note about embedding versus external linking
+!! Two ways to embed a PDF in your TW5 file.
 
-''Note:'' Before we dive into how you can embed files of a certain type, you may want to consider not embedding and instead simply linking to external resources like this:
+''Note:'' Before embedding files in TiddlyWiki, you may want to consider instead simply linking to the external resource like this:
 
 ```
 [ext[title|path/to/that.file]] 
 ```
 
 Then let your browser decide how to handle the resource. The advantage to this approach is that it is simple and quick. The disadvantage is that the file doesn't actually travel with the TW5 file and that you can't use `node.js` with method #2 below to quickly file paths.
 
-!! Two ways to embed a PDF in your TW5 file.
-
 !!! 1. Embedding with Drag and Drop 
 
-One is to drag and drop the file onto your TW5 file. Wait for the green bar to appear at the top, drop, and then import from the import tiddler. This actually embeds the contents of your imported PDF (or other file) into your TW5 file.
+Drag and drop the file onto your TiddlyWiki or use other methods to [[import external files|Importing Tiddlers]]. 
+
+This method is not recommended for larger files because they add to the total size and load time for your TiddlyWiki file.
 
-This method be OK as long as your PDF is not too big. There can be concerns if your TW5 file becomes too big.
+''Note:'' Some browsers, most notably IE/Edge can have problems to display embedded PDF files.
 
 !!! 2. Embedding with '_canonical_uri'
 
-The other way is to create a tiddler link to the external file. In this method the file is not actually incorporated into your TW5 file, but can be accessed with the `{{My Image File.jpg}}` transclusion syntax just like an embedded file. The location address of the file can also be changed under [[node.js]]. See [[ExternalImages]] for details of using external images with node.js.
```

Comment:
TBH I can't remember my reasoning so if you want it there, just keep it of course. However, it seems to me that "the location address of the file can also be changed under node.js" is out of scope of the tiddler title. The text is a bit all over the place.

Maybe the tiddler should be re-made into an umbrella tiddler titled "How to embed external files" with links to specific use case tiddlers and methods, where one treats the _canonical_uri field with a subheading about the node.js aspects.